### PR TITLE
Fix the Sign Up button issue of the consent only provisioning flow

### PR DESCRIPTION
**Pull request by DilshanSenarath**
### Related Issues
- https://github.com/wso2/product-is/issues/19674

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by DilshanSenarath** [comment]
https://github.com/wso2/identity-apps/assets/74205483/2c7b76ed-cd0e-403f-a635-825e35644fca

File: identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-with-verification.jsp [hunk 1]
```diff
@@ -891,53 +892,55 @@
...
-                + " " + passwordConfig.minNumber + " "
-                + '<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "numbers")%>';
-            $("#number-block").css("display", "block");
-        }
-        if ((passwordConfig.minUpperCase > 0) || passwordConfig.minLowerCase > 0) {
-            let cases = [];
-            if (passwordConfig.minUpperCase > 0) {
-                cases.push(passwordConfig.minUpperCase + " "
-                    + '<%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "uppercase")%>');
+        if (passwordValidationBlock.length > 0) {
```
Comment (pavinduLakshan) [inline identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-with-verification.jsp:895]: Won't this throw a runtime error such as `Cannot read length of undefined`, if the passwordValidationBlock is not found?

Comment (DilshanSenarath) [reply identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-with-verification.jsp:895]: Internally, jQuery handles this process. If there is no matching element, jQuery will return an empty object. To check the existence of an element, it is recommended to use the condition `if (passwordValidationBlock.length > 0)`.

**Review by pavinduLakshan — APPROVED** [review]

File: identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-with-verification.jsp (hunk 1 above)
Comment (pavinduLakshan) [reply identity-apps-core/apps/recovery-portal/src/main/webapp/self-registration-with-verification.jsp:895]: Ah nice

**Pull request merged**